docs/extensions/METADATA.md

```diff
@@ -80,7 +88,30 @@ For more advanced use cases, the extension can also:
 },
 ```
 
-The vm metadata section should define either `image` or `composefile`. When you use `image`, a default compose file is generated for the extension.
+The composefile would then look like (for exampe with a volume definition):
+
+```yaml
+services:
+  myExtension:
+    image: ${DESKTOP_PLUGIN_IMAGE}
+    volumes:
+      - /host/path:/container/path
+```
+
+### Using the docker socket from your extension backend
+
+Docker extensions can invoke Docker commands directly from the UI js API. In some cases it can be useful to also interact with the Docker engine from the backend. Extension backend containers can mount the docker socket and use it to interact with the Docker engine from the extension backend logic.
```

Comment:
Shouldn't we explain what the docker socket is for developers that don't know how docker engine and docker desktop internals work?

Reply:
I would assume developers who don't know what it is will likely not need this level of information, and need to learn more about Docker in general in order to develop complex extensions. (or they would just execute the docker CLI from the SDK for simple use cases)
I'm not sure if we should point them at some general Docker tutorial... I would assume if you read this you already know about the general Docker mechanisms